<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькулятор похідних</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Функція:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="inputFunction">
       <property name="inputMethodHints">
        <set>Qt::ImhLowercaseOnly</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Змінна диф.:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="inputVariable"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Похідна:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="outputDerivative">
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Обчислити</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>inputFunction</tabstop>
  <tabstop>inputVariable</tabstop>
  <tabstop>calculateButton</tabstop>
  <tabstop>outputDerivative</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>calculateButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>calculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calculate()</slot>
 </slots>
</ui>
